docs/intro/contributing.txt

```diff
@@ -54,7 +54,7 @@ By the end of this tutorial, you should have a basic understanding of both the
 tools and the processes involved. Specifically, we'll be covering the following:
 
 * Installing Git.
```

Comment:
As noted by @IgnatiusFlo, it's a bit confusing what level this tutorial is aiming at, especially since Git is much more widespread these days - most people comfortable enough with the command line and python to complete the tutorial are likely to already have run a few git commands.